src/walletinitinterface.h

```diff
@@ -5,6 +5,8 @@
 #ifndef WALLETINITINTERFACE_H
 #define WALLETINITINTERFACE_H
 
+#include <string>
```

Comment:
In commit "[wallet] Use global g_wallet_init_interface to init/destroy the wallet."

This change would make a little more sense in the previous commit adding the file.

Reply:
Agree. Fixed